pkg/platform/model/buildplanner/impactreport.go

```diff
@@ -27,9 +32,26 @@ func (b *BuildPlanner) ImpactReport(params *ImpactReportParams) (*response.Impac
 		return nil, errs.Wrap(err, "Unable to marshal buildexpression")
 	}
 
-	request := request.ImpactReport(params.Owner, params.Project, beforeExpr, afterExpr)
+	var resp *response.ImpactReportResponse
+	for i := 0; i < numRetries; i++ {
+		resp, err = run(b.client, params.Owner, params.Project, beforeExpr, afterExpr)
+		if err == nil || !response.IsImpactReportBuildPlanningError(err) {
+			break
+		}
+		logging.Debug("Impact report response was that the buildplanner was still planning; trying again")
+		time.Sleep(retryInterval * time.Duration(i+1))
```

Comment:
```suggestion
		time.Sleep(retryInterval * (i+1))
```

I don't think you need `time.Duration()`, and it was a bit confusing to read.

Reply:
Unfortunately I do. I tried `retryInterval * (i+1)` first, but got a compiler error. Google says I need to cast to `time.Duration()`. I don't like it, but what can you do?

Reply:
As discussed in stand, we need a cast to satisfy the compiler. I did use a separate variable in an attempt to improve readability, though.